helloworld.py

```diff
@@ -3,4 +3,5 @@
 """
 
 if __name__ == '__main__':
-    print('Hello, World!')
\ No newline at end of file
+    user_name = input('What is your name? ')
+    print('Hello ' + user_name + '!')
```

Comment:
파이썬에 익숙하지 않은 분들을 위해 주석을 추가하는 건 어떨까요?
```suggestion
    # 사용자의 이름을 입력 받는다
    user_name = input('What is your name? ')
    # 사용자의 이름과 함께 인사를 출력한다
    print('Hello ' + user_name + '!')
```